web-marketplace/src/pages/CreditClassDetails/CreditClassDetailsSimple/CreditClassDetailsSimple.tsx

```diff
@@ -97,6 +97,7 @@ const CreditClassDetailsSimple: React.FC<
 
```

Comment:
not related to the changes in this PR, but looks like CREDIT_CLASS_TOOLTIP isn't getting extracted

Reply:
I rerun the extract, and it is now.